modules/claims_api/config/initializers/blueprinter.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+Blueprinter.configure do |config|
+  config.sort_fields_by = :definition
+end
```

Comment:
this is required to keep the order as they're defined. If we remove this, they go back to alphabetized, which changes from v1 a bit but probably isn't a concern. Thinking we discuss this in standup. Probably not essential.